docker/oso-monitor-sso/src/root/templates/zagg_client.yaml.j2

```diff
@@ -0,0 +1,27 @@
+heartbeat:
+    hostgroups:
+    - ops-aws
+    - prod
+    templates:
+    - Template Heartbeat
+    - Template Ops SSO
+host:
+    name: {{ hostname }}
+pcp:
+    metrics:
+    - sso.iam.not.reachable
+synthetic_clusterwide:
+    heartbeat:
+        templates:
+        - Template Heartbeat
+        - Template OpenShift Cluster
+        - Template Ops SSO
+    host:
+        name: ops-aws-synthetic
```

Comment:
I have no idea what this synthetic_clusterwide section is for, so I just left it alone. you can tweak it if needed. I added the hostname in above in the normal host: section